routes.js

```diff
@@ -0,0 +1,64 @@
+const express = require('express');
+const router = express.Router();
+const low = require('lowdb');
+const fileAsync = require('lowdb/lib/storages/file-async');
+const db = low('db/db.json', {
+  storage: fileAsync
+});
+
+router.get('/movies', (req, res) => {
```

Comment:
Because all the routes in this router are prepended with `/movies` it would be preferrable to use `app.use("/movies", router)`; in `app.js` and `router.get("/", ....` here.